src/test/java/org/cactoos/text/UncheckedTextTest.java

```diff
@@ -43,4 +45,16 @@ public void rethrowsCheckedToUncheckedException() {
         ).asString();
     }
 
+    @Test
+    public void toStringMustReturnSameOfAsString() {
+        final String text = "one";
+        new Assertion<>(
+            "Must implement #toString which returns the same of #asString",
+            new UncheckedText(
+                new TextOf(text)
+            ).toString(),
```

Comment:
@fabriciofx 
You can use `HasToString` here, instead of calling the method directly.